### DISCUSSION TITLE
Отправка HTTP-запросов ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#41] [Announcements]
# Отправка HTTP-запросов ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/guides/webdraftt/http-requests/

**Comment by DoWh** [comment]
request устарел

**Comment by YuraKissil** [comment]
Прошу исправить "Метод get() принимает аргументами URL" -> Метод request() принимает аргументами URL
